.github/workflows/test.yml

```diff
@@ -135,8 +135,8 @@ jobs:
     - name: ESP32 Build & Clippy
       if : needs.changes.outputs.src-esp32 == 'true' || needs.changes.outputs.src-common == 'true' ||  needs.changes.outputs.src-servers == 'true'
       run: |
-        bash -c 'git config --global --add safe.directory /opt/esp/esp-idf && . "$IDF_PATH"/export.sh && . "$ESP_ROOT"/export-esp.sh && make clippy-esp32'
-        bash -c '. "$IDF_PATH"/export.sh && . "$ESP_ROOT"/export-esp.sh && cd micro-rdk-server && cd ../ && make build-esp32-bin'
+        bash -c '. /home/testbot/.bash_profile ; git config --global --add safe.directory "$ESP_ROOT"/esp-idf && make clippy-esp32'
+        bash -c '. /home/testbot/.bash_profile ; cd micro-rdk-server && cd ../ && make build-esp32-bin'
```

Comment:
is the second and third command on this line necessary?

Reply:
Yeah that's a little odd now that you point it out. I hadn't noticed it. As you said, it isn't changed here, but it does seem to amount to a no-op. I'll remove it if I revisit this file.